dist/src/main/resources/schema.sql

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+-- Ref: https://docs.spring.io/spring-security/reference/servlet/appendix/database-schema.html#_user_schema
+CREATE TABLE IF NOT EXISTS users (
```

Comment:
Apparently the schema file is now in `identity/common` (https://github.com/camunda/camunda/blob/93b38edf9bbaa79911f69df6d6904caa9a62bf2f/identity/common/src/main/resources/schema.sql), so it will be moved up?

Reply:
Yea the schema file before this PR was in Identity-common but I moved it here because of it being relevant to all components (I'm happy to move it back though in my PR), either way I will rebase it based on Maryam's PR after she merges

Reply:
The current schema should then be removed, right? https://github.com/camunda/camunda/blob/main/identity/common/src/main/resources/schema.sql From the changes in this PR it looks like both files are still present and this one is not up to date after [the](https://github.com/camunda/camunda/pull/18773) was merged

Reply:
❓ Is the schema file automatically included via it's path `main/resources/schame.sql`?

Reply:
Exactly, this is a known path to Spring so the file is automatically picked up